core/src/blockchain/extras.rs

```diff
@@ -93,10 +93,10 @@ impl Key<TransactionAddresses> for Tracker {
 /// Familial details concerning a block
 #[derive(Debug, Clone, RlpEncodable, RlpDecodable)]
 pub struct BlockDetails {
+    /// Block view
+    pub view: u64,
```

Comment:
Though It is just a type alias but what about using `View` type defined in `core/types`